package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "@arcadia-eng/utility-connect-react",
-  "version": "4.0.3",
-  "description": "React tool for integrating with Arcadia's utility connect service",
+  "name": "@arcadia-eng/connect-react",
+  "version": "5.0.0",
+  "description": "React tool for integrating with Arc Connect",
```

Comment:
Technically, `Arc Connect` is the name of the React tool, no? Should this be "React component for integrating with Arc Plug"? 😕 @johnnyman727

Reply:
"React package for implementing Arc Connect" maybe? In my mind the name of the widget is `Connect` but it's part of the Arc ecosystem so it is situationally appropriate to preface it with that